src/extensions/bigbang/test/bigbang_test.go

```diff
@@ -45,43 +47,52 @@ func TestMain(m *testing.M) {
 
 func TestReleases(t *testing.T) {
 	// Initialize the cluster with the Git server and AMD64 architecture
-	zarfExec(t, "init", "--confirm", "--components", "git-server", "--architecture", "amd64")
+	arch := "amd64"
+	stdOut, stdErr, err := zarfExec(t, "init", "--confirm", "--components", "git-server", "--architecture", arch)
+	require.NoError(t, err, stdOut, stdErr)
+
+	// Remove the init package to free up disk space on the test runner
+	_ = os.RemoveAll(fmt.Sprintf("zarf-init-%s-%s", arch, getZarfVersion(t)))
```

Comment:
This is not the correct location for the init package.

Reply:
Fixed to add `tar.zst` suffix to the init package